docs/api/wrapper/trigger.md

```diff
@@ -38,6 +38,10 @@ test('trigger demo', async () => {
 })
 ```
 
+::: tip
+When using `trigger('focus')` with [jsdom v16.4.0](https://github.com/jsdom/jsdom/releases/tag/16.4.0) and above you must use the [attachTo](../options.md#attachto) option when mounting the component. This is because `el.focus()` does not work on elements that are disconnected from the DOM.
```

Comment:
Is this a bug in versions of jsdom earlier or later than 16.4.0? Might be good to specify.

Reply:
@lmiller1990 It looks like `jsdom` does consider this change a bug fix. I have updated the tip text to say:

```md
When using `trigger('focus')` with [jsdom v16.4.0](https://github.com/jsdom/jsdom/releases/tag/16.4.0) and above you must use the [attachTo](../options.md#attachto) option when mounting the component. This is because a bug fix in [jsdom v16.4.0](https://github.com/jsdom/jsdom/releases/tag/16.4.0) changed `el.focus()` to do nothing on elements that are disconnected from the DOM.
```

WDYT?

Reply:
Perfect